src/poliastro/czml/extract_czml.py

```diff
@@ -496,28 +285,58 @@ def add_orbit(
             )
 
         self.orbits.append([orbit, N, orbit.epoch])
+        cartesian_cords = self._init_orbit_packet_cords_(self.i)
 
-        self._init_orbit_packet_(self.i)
-
-        self._change_id_params_(
-            self.i, o_id=id_id, name=id_name, description=id_description
-        )
-        self._change_path_params_(
-            self.i,
-            pixel_offset=path_pixel_offset,
-            color=path_color,
-            width=path_width,
-            show=path_show,
+        start_epoch = CZMLExtractor.format_date(
+            min(self.orbits[self.i][2], self.start_epoch)
         )
-        self._change_label_params_(
-            self.i,
-            fill_color=label_fill_color,
-            outline_color=label_outline_color,
-            font=label_font,
-            text=label_text,
-            show=label_show,
+
+        pckt = Packet(
+            id=self.i,
+            name=id_name,
+            description=id_description,
+            availability=start_epoch.value + "/" + self.end_epoch.value,
+            position=Position(
+                interpolationDegree=5,
+                interpolationAlgorithm=InterpolationAlgorithms.LAGRANGE,
+                referenceFrame=ReferenceFrames.INERTIAL,
+                cartesian=cartesian_cords,
+                epoch=start_epoch.value,
+            ),
+            path=Path(
+                show=path_show,
+                width=path_width,
+                material=Material(
+                    solidColor=SolidColorMaterial(color=Color(rgba=path_color))
+                )
+                if path_color is not None
+                else Material(
+                    solidColor=SolidColorMaterial(color=Color(rgba=[255, 255, 0, 255]))
+                ),
+                resolution=120,
+            ),
+            label=Label(
+                text=label_text,
+                font=label_font,
+                show=label_show,
+                fillColor=Color(rgba=label_fill_color)
```

Comment:
The default is already `white` https://github.com/AnalyticalGraphicsInc/czml-writer/wiki/Label#properties so probably we don't need to specify it either?

Reply:
I figured that it would be better for the default to be yellow because it's what the original demo uses and in my opinion it looks more "clear". Same thing I changed the font default: 30pt is way too big for a label and it took up half the view-port

Reply:
Perfect then!